src/main/java/today/sesac/shoutify/post/entity/Post.java

```diff
@@ -0,0 +1,88 @@
+package today.sesac.shoutify.post.entity;
+
+import jakarta.persistence.*;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import today.sesac.shoutify.global.domain.BaseEntity;
+
+/**
+ * 게시판의 게시글을 관리하는 도메인입니다.
+ */
+@Entity //엔티티임을 표시
+@Table(name = "posts") //엔티티<->매핑할 테이블 , 이름 생략시 엔티티 이름 사용, 유니크 제약조건
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED) //매개변수 없는 생성자를 protected로 만들어줌
+
+public class Post extends BaseEntity {
+
+    @Id //기본키
+    @GeneratedValue(strategy = GenerationType.IDENTITY) //auto_increment
+    private Long id;
+    /**
+     * 비속어 ai 수정 전 게시물 내용입니다.
+     */
+    @Column(nullable = false, columnDefinition = "TEXT")
+    private String afterContents;
+    /**
+     * 게시물을 작성한 회원입니다
+     */
+//    @ManyToOne
+//    @JoinColumn(name = "author", nullable = false)
+//    private User user;
+    /**
+     * 게시물의 제목입니다.
+     */
+    @Column(length = 255)
+    private String title;
+    /**
+     * 게시물 작성후 ai로 판단되는 게시물에 대한 점수입니다.
+     */
+    @Column(nullable = false, columnDefinition = "INT DEFAULT 0")
+    private int score = 0;
```

Comment:
P3
이것도 그냥 참고만 해 주세요.
int 타입이기 때문에, 이미 INT NOT NULL 이고, default 값은 @ColumnDefault 이걸로 설정 가능합니다.

나중에 테이블을 이미 생성해 놓은 뒤면, 사실 Column은 꼭 필요한 건 아니라서 이건 그냥 참고로 말씀드려요.
```suggestion
    @ColumnDefault("0")
    private int score = 0;
```